exchanges/websocket/monitor/monitor.go

```diff
@@ -95,15 +93,15 @@ func (w *Websocket) Connect() error {
 	go w.trafficMonitor(&anotherWG)
 	anotherWG.Wait()
 	if !w.connectionMonitorRunning {
-		go w.wsConnectionMonitor()
+		go w.connectionMonitor()
 	}
 	go w.manageSubscriptions()
```

Comment:
I know the original work for this was done outside of the PR but manageSubscriptions() & WebsocketReset() on line 226
can return an error that is unhandled

Within both functions _something_ is done such_ as manageSubscriptions() sending it back to the websocket datahandler but the return result is unhandled

Reply:
For clarification, given that go routines do not return the error, it seems like you're asking for me to remove the err in the return, rely on datahandler err channel communication and not remove them as goroutines. Is this correct?